includes/emails/class-wp-job-manager-email-employer-expiring-job.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * Email notification to employers when a job is updated.
+ *
+ * @package wp-job-manager
+ * @since 1.31.0
+ * @extends WP_Job_Manager_Email
+ */
+class WP_Job_Manager_Email_Employer_Expiring_Job extends WP_Job_Manager_Email_Template {
+	const SETTING_NOTICE_PERIOD_NAME = 'notice_period_days';
+	const SETTING_NOTICE_PERIOD_DEFAULT = '1';
+
+	/**
+	 * Get the unique email notification key.
+	 *
+	 * @return string
+	 */
+	public static function get_key() {
+		return 'employer_expiring_job';
+	}
+
+	/**
+	 * Get the friendly name for this email notification.
+	 *
+	 * @return string
+	 */
+	public static function get_name() {
+		return __( 'Employer Notice of Expiring Job Listings', 'wp-job-manager' );
+	}
+
+	/**
+	 * Get the description for this email notification.
+	 *
+	 * @type abstract
+	 * @return string
+	 */
+	public static function get_description() {
+		return __( 'Send notices to employers before a job listing expires.', 'wp-job-manager' );
+	}
+
+	/**
+	 * Get the notice period in days from the notification settings.
+	 *
+	 * @param array $settings
+	 * @return int
+	 */
+	public static function get_notice_period( $settings ) {
+		if ( isset( $settings[ self::SETTING_NOTICE_PERIOD_NAME ] ) ) {
+			return absint( $settings[ self::SETTING_NOTICE_PERIOD_NAME ] );
+		}
+		return absint( self::SETTING_NOTICE_PERIOD_DEFAULT );
+	}
+
+	/**
+	 * Get the email subject.
+	 *
+	 * @return string
+	 */
+	public function get_subject() {
+		$args = $this->get_args();
+
+		/**
+		 * @var WP_Post $job
+		 */
+		$job = $args['job'];
+		return sprintf( __( 'Job Listing Expiring: %s', 'wp-job-manager' ), $job->post_title );
+	}
+
+	/**
+	 * Get `From:` address header value. Can be simple email or formatted `Firstname Lastname <email@example.com>`.
+	 *
+	 * @return string|bool Email from value or false to use WordPress' default.
+	 */
+	public function get_from() {
+		return false;
+	}
+
+	/**
+	 * Get array or comma-separated list of email addresses to send message.
+	 *
+	 * @return string|array
+	 */
+	public function get_to() {
+		$args = $this->get_args();
+		return $args['author']->user_email;
+	}
+
+	/**
+	 * Expand arguments as necessary for the generation of the email.
+	 *
+	 * @param $args
+	 * @return mixed
+	 */
+	protected function prepare_args( $args ) {
```

Comment:
Just a thought (not blocking for this PR):

All three email subclasses so far have the `job_id` logic in this method, and two of them have the `user_id` logic as well. I wonder if it would be worth writing a couple of helper methods so that this logic isn't duplicated? I know it's not a lot of code, but it seems like it could end up being copied in a lot of subclasses.